### PR TITLE
Convert to gradle

### PR DESCRIPTION
**Pull request by applitect**
- Rewrite the build framework to go from sbt to gradle. 
- Update to release with Java 11.
- Minor javadoc fixes to allow it to compile with Java 11.

Has been tested and have been able to deploy a snapshot version of 3.8 to sonatype oss. Deploys in the same structure as the sbt deploy.

### CODE REVIEW

File: connectors/activemq/build.gradle [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies {
+    api project(':forklift-core'),
+            'org.apache.activemq:activemq-client:5/14.0',
```
Comment (Kuroshii) [inline connectors/activemq/build.gradle:3]: Should this be `5.14.0`?

Comment (applitect) [reply connectors/activemq/build.gradle:3]: Probably. I bet in came in transitively from plug-in. Going to look to see if I can remove it.

Comment (applitect) [reply connectors/activemq/build.gradle:3]: I bet bringing in the broker brought in the client correctly. But fixing.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,6 @@
-language: scala
-scala:
-- 2.11.4
+language: java
 jdk:
-- oraclejdk8
```
Comment (dcshock) [inline .travis.yml:5]: Let's be sure to note this in the release notes. "No longer support jdk8"

File: connectors/activemq/build.gradle [hunk 3]
```diff
@@ -0,0 +1,9 @@
+dependencies {
+    api project(':forklift-core'),
+            'org.apache.activemq:activemq-client:5.14.0',
+            'org.apache.activemq:activemq-broker:5.14.0',
```
Comment (dcshock) [inline connectors/activemq/build.gradle:4]: Do we want to push the broker code to everyone that uses the connector library?

File: server/build.gradle [hunk 4]
```diff
@@ -0,0 +1,34 @@
+apply plugin: 'java'
+apply plugin: 'application'
```
Comment (dcshock) [inline server/build.gradle:2]: Does this build the same binary as the sbt version?

File: .travis.yml (hunk 2 above)
Comment (applitect) [reply .travis.yml:5]: Already got another branch going for updating docs for the next release.

File: connectors/activemq/build.gradle (hunk 3 above)
Comment (applitect) [reply connectors/activemq/build.gradle:4]: Good question. Let me look at that.

File: server/build.gradle (hunk 4 above)
Comment (applitect) [reply server/build.gradle:2]: Yes, it builds the same binary and creates a shell script. Not identical shell script but it works the same.

**Pull request merged**